.github/workflows/test.yml

```diff
@@ -115,8 +113,29 @@ jobs:
           path: ${{ github.workspace }}/artifacts
           retention-days: 1
 
+      - name: "(Windows) Test building C bindings - dynamic link vc runtime"
+        if: ${{ matrix.platform == 'windows-latest' }}
+        shell: bash
```

Comment:
It never ceases to amaze me that there is now supported bash on windows

Reply:
Yeah Git bash is installed :) Not really conventional but it works 🤷